http/codegen/server.go

```diff
@@ -1352,6 +1352,9 @@ const responseT = `{{ define "response" -}}
 			{{- if .HTTPOnly }}
 			HttpOnly: true,
 			{{- end }}
+			{{- if .SameSite }}
+			SameSite: {{ printf "%q" .SameSite }},
```

Comment:
Will these generated lines print the enum from the `net/http` package e.g. `http.SameSiteLaxMode`? It just occurred to me that this could be a problem, as `http.SameSite_` are iotas.

Reply:
Actually there are a couple of issues with this including the `"%q"` 😅 Will fix